python/lang/correctness/sync-sleep-in-async-code.yaml

```diff
@@ -0,0 +1,18 @@
+rules:
+- id: sync-sleep-in-async-code
+  patterns:
+  - pattern: time.sleep(...)
+  - pattern-inside: |
+      async def $F(...):
+        ...
+  - pattern-not-inside: |
+      async def $F(...):
+        def $INNER(...):
+          ...
+  message: Synchronous sync in async ccode will block the event loop. Use asyncio.sleep() instead
```

Comment:
DId you mean "Synchronous time.sleep(...)"?

Reply:
fixed